introduction.md

```diff
@@ -19,6 +19,20 @@ xx However, not promising and yet not speechless. [有為有守 - ]
 
 洞山五位臨濟三玄。共是垂慈心。聲止啼黃葉。若就語中生實解者。求馬於唐肆待兔於枯株也。
 
+DongShan Mountain (note, dong is cave/hole/zero)
+Linji, you may know.
+
+DongShan Mountain ;五位; Linji Sanxuan. 
+  
+xxThere is always kindness. 
+  
+xxThe sound stops crying yellow leaves.
```

Comment:
sound/voice stop, weep of yellow leaves
dying leaves of autumn or walking on dry leaves vibe